image/v3/modifying/text-fonts.md

```diff
@@ -85,7 +85,7 @@ Set a path to a font file in file system in which the text should be written.
 
 ### Color
 
-> public FontInterface::color(mixed $color): FontInterface
+> public FontInterface::setColor(mixed $color): FontInterface
```

Comment:
Please revert. This is correct call of class Intervention\Image\Typography\FontFactory::class.